server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds.Contracts/GetLatestApplicationBuildPath.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Arcadia.Assistant.AppCenterBuilds.Contracts
+{
+    public class GetLatestApplicationBuildPath
```

Comment:
There is no Akka anymore, do not use messages

Reply:
Done